device/bgp/bgp.yaml

```diff
@@ -38,43 +61,31 @@ components:
         as_number_width:
           description: >-
             The width in bytes of the as_number values.
-            Any as_number values that exceed the width MUST result in an error
+            Any as_number values that exceed the width MUST result in an error.
           type: string
           enum:
           - two
           - four
           default: four
-        as_number_set_mode:
-          description: >-
-            The AS number set mode
-          type: string
-          enum:
-          - do_not_include_as
-          - include_as_seq
-          - include_as_set
-          - include_as_seq_confed
-          - include_as_set_confed
-          - prepend_as_to_first_segment
-          default: do_not_include_as          
         advanced:
-          $ref: './bgpadvanced.yaml#/components/schemas/Device.BgpAdvanced'
+          $ref: './bgpadvanced.yaml#/components/schemas/Bgp.Advanced'
         capability:
-          $ref: './bgpcapability.yaml#/components/schemas/Device.BgpCapability'
+          $ref: './bgpcapability.yaml#/components/schemas/Bgp.Capability'
         sr_te_policies:
           description: >-
-            Segment routing/traffic engineering policies
+            Segment routing/traffic engineering policies.
           type: array
           items:
-            $ref: './bgpsrtepolicy.yaml#/components/schemas/Device.BgpSrTePolicy'
-        bgpv4_routes:
+            $ref: './bgpsrtepolicy.yaml#/components/schemas/Bgp.SrTePolicy'
+        v4_routes:
           description: >-
-            Emulated BGPv4 routes
+            Emulated BGPv4 route ranges.
```

Comment:
Array of BGPv4 route ranges

Reply:
Type is already array.  So, I think its ok.